glue/.htaccess

```diff
@@ -0,0 +1,29 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+
+## CONTENT NEGOTIATION
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml) [OR]
```

Comment:
You should remove the `[OR]` on this line.  This would be true for all the types you are trying to check for below.